prow/cmd/checkconfig/main_test.go

```diff
@@ -1252,10 +1252,12 @@ func TestValidateInRepoConfig(t *testing.T) {
 			prowYAMLFileName = tempFile.Name()
 		}
 
-		cfg := &config.Config{
-			ProwConfig: config.ProwConfig{PodNamespace: "my-ns"},
+		// Load an empty config so the defaulting is used
+		cfg, err := config.Load("/dev/null", "")
```

Comment:
I think it would be better to create a temporary empty file instead of using `/dev/null` since that won't work on Windows. I'm not sure if our binaries are already making assumptions about running on linux/macos though.
/shrug